src/main/java/com/minecolonies/coremod/colony/colonyEvents/raidEvents/pirateEvent/ShipSize.java

```diff
@@ -1,42 +1,33 @@
 package com.minecolonies.coremod.colony.colonyEvents.raidEvents.pirateEvent;
 
 import com.minecolonies.api.util.constant.ColonyConstants;
-import net.minecraft.entity.EntityType;
-
-import static com.minecolonies.api.entity.ModEntities.*;
 
 /**
  * Enum for ship sizes.
  */
 public enum ShipSize
 {
-    SMALL(ColonyConstants.SMALL_HORDE_SIZE, ColonyConstants.SMALL_PIRATE_SHIP, ColonyConstants.SMALL_HORDE_MESSAGE_ID, 1, PIRATE, PIRATE),
+    SMALL(ColonyConstants.SMALL_HORDE_SIZE, ColonyConstants.SMALL_SHIP, ColonyConstants.SMALL_HORDE_MESSAGE_ID, 2, 2, 0, 0),
```

Comment:
increased from 2 normal to 2 normal 2 archers, think 1 normal 1 archer fits best

Reply:
the first 2 is for the number of spawners

Reply:
you're right nvm^^